src/plopp/widgets/slice.py

```diff
@@ -112,3 +140,23 @@ def slice_dims(data_array: sc.DataArray, slices: Dict[str, slice]) -> sc.DataArr
     for dim, sl in slices.items():
         out = out[dim, sl]
     return out
+
+
+@node
+def range_slice_dims_sum(
+    data_array: sc.DataArray, slices: Dict[str, slice]
+) -> sc.DataArray:
+    """
+    Slice the data according to input slices and sum over the sliced dimensions.
+
+    Parameters
+    ----------
+    data_array:
+        The input data array to be sliced.
+    slices:
+        Dict of slices to apply for each dimension.
+    """
+    out = data_array
+    for dim, sl in slices.items():
+        out = out[dim, slice(*sl)].sum(dim=dim)
```

Comment:
In my mind, the slider would just slice. It would then be up to the user to decide what to do after, possibly using another node to do the `sum` or `mean`.
So I would just return the sliced data.

Maybe we can then make a single `slice_dims` function that would work with both a single integer for slicing, as well as a range?

Reply:
I think returning the slice and asking the user to add a new node to do the operation could be fine. PointCloud requires a single dimension and we need to do something with the returned slice to plot it out.